### PR TITLE
Replace links from MSDN to docs.microsoft.com

### PR DESCRIPTION
**Pull request by iSazonov**
# PR Summary

Replace links from MSDN to docs.microsoft.com. Old links died.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 1]
```diff
@@ -328,14 +328,14 @@ For more details, see [Create and Run PowerShell Script Guide][create-run-script
 [ebook-list]: https://martin77s.wordpress.com/2014/05/26/free-powershell-ebooks/
 [tutorial]: https://www.computerperformance.co.uk/powershell/index.htm
 [to-learn]:https://blogs.technet.microsoft.com/heyscriptingguy/2015/01/04/weekend-scripter-the-best-ways-to-learn-powershell/
-[ps-module]:https://msdn.microsoft.com/library/dd878324%28v=vs.85%29.aspx
+[ps-module]:https://docs.microsoft.com/en-us/powershell/scripting/developer/module/understanding-a-windows-powershell-module
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/powershell-beginners-guide.md]: Locale needs to be removed

```suggestion
[ps-module]:https://docs.microsoft.com/powershell/scripting/developer/module/understanding-a-windows-powershell-module
```

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 2]
```diff
@@ -328,14 +328,14 @@ For more details, see [Create and Run PowerShell Script Guide][create-run-script
...
 [create-ps-module]:https://www.business.com/articles/powershell-modules/
 [remoting]:https://channel9.msdn.com/Series/GetStartedPowerShell3/06
 [in-depth]: https://channel9.msdn.com/events/MMS/2012/SV-B406
 [remote-mgmt]:https://www.itprotoday.com/powershell/powershell-basics-remote-management
-[remote-commands]:https://docs.microsoft.com/powershell/scripting/core-powershell/running-remote-commands?view=powershell-6
+[remote-commands]:https://docs.microsoft.com/powershell/scripting/core-powershell/running-remote-commands
 [examples]:https://examples.oreilly.com/9780596528492/
-[examples-ps-module]:https://msdn.microsoft.com/library/dd878340%28v=vs.85%29.aspx
+[examples-ps-module]:https://docs.microsoft.com/en-us/powershell/scripting/developer/module/how-to-write-a-powershell-script-module
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/powershell-beginners-guide.md]:
```suggestion
[examples-ps-module]:https://docs.microsoft.com/powershell/scripting/developer/module/how-to-write-a-powershell-script-module
```

File: docs/learning-powershell/powershell-beginners-guide.md [hunk 3]
```diff
@@ -328,14 +328,14 @@ For more details, see [Create and Run PowerShell Script Guide][create-run-script
...
 [writing-ps-module]:https://www.powershellmagazine.com/2014/03/18/writing-a-powershell-module-in-c-part-1-the-basics/
-[sample-code]:https://msdn.microsoft.com/library/ff602031%28v=vs.85%29.aspx
+[sample-code]:https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/examples-of-cmdlet-code
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/powershell-beginners-guide.md]:
```suggestion
[sample-code]:https://docs.microsoft.com/powershell/scripting/developer/cmdlet/examples-of-cmdlet-code
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by vexx32** [comment]
Oh, _that's_ what happened. Weird. Thanks for fixing this, @iSazonov! 💖

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@iSazonov let's merge this so we can unblock other PRs

**Comment by iSazonov** [comment]
Done.

**Pull request merged**